.github/workflows/update_robottelo_image.yml

```diff
@@ -0,0 +1,88 @@
+name: update_robottelo_image
+
+on: [push]
```

Comment:
Since we accept PRs to mutliple branches, I would be more specific here that this is on master branch only.

I think we're going to want separate image labels for the z-stream branches, and a separate workflow for them.

```
on:
  push:
    branches:
      - master
```

Reply:
Actually I see you've handled this with the git ref setting the image tag, well done.

Reply:
yes